src/components/Card/CardContent.js

```diff
@@ -1,18 +1,24 @@
 // @flow
 import React from 'react'
-import styled from 'styled-components'
+import styled, {css} from 'styled-components'
 import extractDomain from 'extract-domain'
 
 type ContentProps = {
   title?: string,
   description?: string,
-  url?: string
+  url?: string,
+  large?: boolean
 }
 
-const ContentWrap = styled.div`
+const CardContent = styled.div`
   flex: 1;
   padding: 10px 15px;
   min-width: 0;
+  box-sizing: border-box;
+
+  ${props => props.large && css`
```

Comment:
this is the styled-components way to add conditional styles

Reply:
Yeah thanks, I didn't want to have to give a prop to the 2 components though, cause this also needs to be passed to the emptyState (in #9). But we'll see how it goes